clang/lib/Parse/ParseCilk.cpp

```diff
@@ -428,21 +437,21 @@ StmtResult Parser::ParseCilkForStatement(SourceLocation *TrailingElseLoc) {
   // // We need to perform most of the semantic analysis for a C++0x for-range
   // // statememt before parsing the body, in order to be able to deduce the type
   // // of an auto-typed loop variable.
```

Comment:
These comments are now relevant for for-range statements.  Please remove the extra `// ` on these comments.